pos_save_client_error_fix/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+POS Save Client Error Message FIX
+==============
+
+This module extends the functionality of POS and displays the error message returned by the server rather than "Your Internet connection is probably down.".
+
```

Comment:
I've tested in runbot and I get the message "You Internet connection is probable down" so I don't know what to expect with this module.

Could you please explain in the "Usage" seccion how to reproduce the functionality of the module please?

![2016-12-14_6-15-46](https://cloud.githubusercontent.com/assets/8736623/21170379/fa94d5ae-c1c4-11e6-8452-5e34af174617.png)